app.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const handlebars = require('express-handlebars').create({
+  defaultLayout: 'main',
+});
+const bodyParser = require('body-parser');
+const cookieParser = require('cookie-parser');
+const session = require('express-session');
+
+const app = express();
+app.engine('handlebars', handlebars.engine);
+
+app.set('port', process.argv[2] || 4231);
+app.set('view engine', 'handlebars');
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+app.use(cookieParser());
+app.use(session({secret: "Yz1phIjvWxQrSt90dEf2UKlbC678MnAO"}));
```

Comment:
I don't think we'd need sessions if we're using Google SignIn. Google signin uses OAuth protocol, not sessions

Reply:
But I'm going to accept and merge anyway since I want to get started on my piece

Reply:
Thank you. I wasn't sure how sessions worked; I just knew we went over it briefly in my web dev class. I have removed sessions as a new commit on this pull request.